README.md

```diff
@@ -93,34 +54,44 @@ docker run -v /path/to/config/dir:/config \
+
+```shell
+martin  /path/to/mbtiles/file.mbtiles
+martin  /path/to/directory
+```
+
+## API
 
 When started, Martin will go through all spatial tables and functions with an appropriate signature in the database. These tables and functions will be available as the HTTP endpoints, which you can use to query Mapbox vector tiles.
 
-| Method | URL                                    | Description                                             |
-|--------|----------------------------------------|---------------------------------------------------------|
-| `GET`  | `/`                                    | Status text, that will eventually show web UI           |
-| `GET`  | `/catalog`                             | [List of all sources](#source-list)                     |
-| `GET`  | `/{sourceID}`                          | [Source TileJSON](#table-source-tilejson)               |
-| `GET`  | `/{sourceID}/{z}/{x}/{y}`              | [Source Tiles](#table-source-tiles)                     |
-| `GET`  | `/{sourceID1},...,{nameN}`             | [Composite Source TileJSON](#composite-source-tilejson) |
-| `GET`  | `/{sourceID1},...,{nameN}/{z}/{x}/{y}` | [Composite Source Tiles](#composite-source-tiles)       |
-| `GET`  | `/health`                              | Martin server health check: returns 200 `OK`            |
-
-# Using with MapLibre
+| Method | URL                                    | Description                                                                                               |
+|--------|----------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `GET`  | `/`                                    | Status text, that will eventually show web UI                                                             |
+| `GET`  | `/catalog`                             | [List of all sources](https://maplibre.org/martin/source-list.html)                                       |
+| `GET`  | `/{sourceID}`                          | [Source TileJSON](https://maplibre.org/martin/table-sources.html#table-source-tilejson)                   |
+| `GET`  | `/{sourceID}/{z}/{x}/{y}`              | [Source Tiles](https://maplibre.org/martin/table-sources.html#table-source-tiles)                         |
+| `GET`  | `/{sourceID1},...,{nameN}`             | [Composite Source TileJSON](https://maplibre.org/martin/composite-sources.html#composite-source-tilejson) |
+| `GET`  | `/{sourceID1},...,{nameN}/{z}/{x}/{y}` | [Composite Source Tiles](https://maplibre.org/martin/composite-sources.html#composite-source-tiles)       |
+| `GET`  | `/health`                              | Martin server health check: returns 200 `OK`                                                              |
+
+## Using with clients
```

Comment:
I think all the usage with various clients should be deleted too - lets keep it only in the book.

Reply:
Removed